src/DataProtection/DataProtection/ref/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+﻿<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <NoWarn>$(NoWarn);CS0618</NoWarn>
```

Comment:
Why can't this project use the standard ref props/targets?

Reply:
I might have added this before adding common props.